frontend/apps/reader/modules/readergesture.lua

```diff
@@ -631,6 +631,36 @@ Default value: %1]]), GestureDetector.PAN_DELAYED_INTERVAL/1000),
                     UIManager:show(items)
                 end,
             },
+            {
+                text = _("Hold pan rate"),
+                callback = function()
+                    local SpinWidget = require("ui/widget/spinwidget")
+                    local current_value = G_reader_settings:readSetting("pan_rate")
+                    if not current_value then
+                        current_value = Screen.low_pan_rate and 2.0 or 30.0
+                    end
+                    local items = SpinWidget:new{
+                        text = T(_([[
+Used when highlighting text, bigger value faster refresh and bigger CPU usage.
+5.0 should be good for most e-ink ereaders.
```

Comment:
If so, why is 2 the default?

Reply:
To be fair, that's a good question. It was default in `readerhighlight.lua` so I kept it that way. I think it's because older devices may have trouble handling higher value, it was kept low just in case. My Kobo Clara HD had no troubles handling 10, but there was no difference between 10 and 5, because of screen refresh(black thingy).

Reply:
In any case, it's weird to say that x is a good value and then not to have x as the default.